Sources/OpenAPIRuntime/Conversion/Configuration.swift

```diff
@@ -105,17 +123,24 @@ public struct Configuration: Sendable {
     /// The generator to use when creating mutlipart bodies.
     public var multipartBoundaryGenerator: any MultipartBoundaryGenerator
 
+    /// Custom XML coder for encoding and decoding xml bodies.
+    public var xmlCoder: (any CustomCoder)?
+    
     /// Creates a new configuration with the specified values.
     ///
     /// - Parameters:
     ///   - dateTranscoder: The transcoder to use when converting between date
     ///   and string values.
     ///   - multipartBoundaryGenerator: The generator to use when creating mutlipart bodies.
+    ///   - xmlCoder: Custom XML coder for encoding and decoding xml bodies.
     public init(
         dateTranscoder: any DateTranscoder = .iso8601,
-        multipartBoundaryGenerator: any MultipartBoundaryGenerator = .random
+        multipartBoundaryGenerator: any MultipartBoundaryGenerator = .random,
+        xmlCoder: (any CustomCoder)? = nil
```

Comment:
You'll also need to add a deprecated version of the 2-param initializer into `Deprecated.swift`, as adding a parameter is technically an API-breaking change, so we also need the previous one, and it'll just call back into this one. Something like:

```swift
extension Configuration {
    /// Creates a new payload.
    @available(*, deprecated, renamed: "init(dateTranscoder:multipartBoundaryGenerator:xmlCoder:)") @_disfavoredOverload
    public init(
        dateTranscoder: any DateTranscoder = .iso8601,
        multipartBoundaryGenerator: any MultipartBoundaryGenerator = .random
    ) {
        self.init(
            dateTranscoder: dateTranscoder,
            multipartBoundaryGenerator: multipartBoundaryGenerator,
            xmlCoder: nil
        )
    }
}
```